concepts/prompt/prompt-api.mdx

```diff
@@ -34,6 +34,65 @@ main();
 
 </CodeGroup>
 
+### Create a prompt
+
+You can create a new prompt on the Literal AI platform using the [Prompt Playground](/concepts/prompt/prompt-playground), but you can also create one using the API. The name of the prompt is its ID, so you cannot change this afterwards.
+
+<CodeGroup>
+```python Python
+import os
+from literalai import LiteralClient
+
+literal_client = LiteralClient(api_key=os.getenv("LITERAL_API_KEY"))
+
+# define prompt
+PROMPT_NAME = "RAG prompt"
+template_messages = [
+    {
+        "role": "system",
+        "content": "You are a helpful assistant that always answers questions. Keep it short, and if available prefer responding with code."
+    },
+    {
+        "role": "user",
+        "content": "Answer the question based on the context below.\nContext:\n{{#context}}\n{{.}}\n{{/context}}\nQuestion:\n{{question}}\nAnswer:"
+    }
+]
+
```

Comment:
For RAG in the context of a chat, we should advertise the correct best practice -> https://github.com/Chainlit/Documentation-RAG-application/blob/main/app/prompts/rag.json

Reply:
Yes, but I'd rather have an example that is best practice. You could do something like
{
        "role": "system",
        "content": "You are a helpful assistant that always answers questions. Keep it short, and if available prefer responding with code. You are talking with {{name}} who has the following interests: {{interests}}"
    }